udfs/README.md

```diff
@@ -45,6 +44,26 @@ or
 
 `CREATE OR REPLACE FUNCTION project_id.your_dataset.function_name() . . .`
 
+### Using JavaScript UDFs
+
+When creating [JavaScript UDFs](https://cloud.google.com/bigquery/docs/reference/standard-sql/user-defined-functions#javascript-udf-structure) in your dataset, you need both to create 
+the UDF and upload the javascript library to a publicly available Google Storage Bucket. 
```

Comment:
```suggestion
the UDF and optionally upload the javascript library to your own Google Storage Bucket. 
```

Reply:
Can you please add another sentence which points out that all JavaScript UDF libraries are currently hosted publicly on gs://bqutil-lib/bq_js_libs/

Reply:
Done